techniques/flash/FLASH29.html

```diff
@@ -72,7 +72,7 @@ <h3>Setting the label using the Component Inspector panel</h3>
       </section>
       <section class="example">
          <h3>Setting the label on a Button, CheckBox and RadioButton component using ActionScript 3.0</h3>
-         <pre xml:space="preserve">import fl.accessibility.ButtonAccImpl
+         <pre>import fl.accessibility.ButtonAccImpl
```

Comment:
all the naked `<pre> ... </pre>` blocks in this technique could do with inner `<code> ... </code>`